### DISCUSSION TITLE
Read extraction

### DISCUSSION THREAD

**Discussion by nemitheasura** [tleonardi/nanocompore#170] [Q&A]
Hi @tleonardi,

imho nanocompore really rocks. I am new to bioinformatics, so my question may seem kinda dumb, but... Is it possible to filter and extract certain read ID's based on the presence/absence of modification in particular site? I've read that nanocompore has a single-read resolution, but I'm not sure how to extract reads of my interest. 

Could you provide any tips?
I would be grateful. 

Best,
N.

**Comment by tleonardi** [comment]
Hi!
Thanks, glad you like it!
It's definitely possible although it requires a bit of work.
First, you need to run nanocompore with the version in the `readid_tracking` branch. This branch essentially saves the read_id in the SampCompDB object so that each data point can be assigned to a specific read.
Then, to do downstream analysis you can use [this](https://github.com/tleonardi/nanocompore_paper_analyses/blob/master/m6acode/parse_sampcompdb.py) script as a reference/starting point. Apologies in advance for the lack of documentation about this.. it's still ongoing work. If you have problems or something is unclear just write here and we'll try to help.
